core/config.js

```diff
@@ -16,28 +16,43 @@ module.exports = class Config {
   // Create config file if it doesn't exist.
   constructor (basePath) {
     this.configPath = path.join(basePath, 'config.json')
+    fs.ensureFileSync(this.configPath)
     this.dbPath = path.join(basePath, 'db')
     fs.ensureDirSync(this.dbPath)
     hideOnWindows(basePath)
-    fs.ensureFileSync(this.configPath)
 
-    if (fs.readFileSync(this.configPath).toString() === '') {
-      this.reset()
-    }
+    this.config = this.read()
 
+    // FIXME: autoBind(this)
```

Comment:
Maybe we can remove this FIXME as it's here since a long time and nobody have feel the need to fix it.

Reply:
Sure thing.